src/CreateMiddleware/CreateMiddleware.php

```diff
@@ -41,10 +41,11 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     /**
      * @param string $class
      * @param string|null $projectRoot
+     * @param string|null $classSkeleton
      * @return string
      * @throws CreateMiddlewareException
      */
-    public function process($class, $projectRoot = null)
+    public function process($class, $projectRoot = null, $classSkeleton = null)
```

Comment:
I agree with that. But it's not really useful to extend the class since its only use would be to add a custom class skeleton. But since it's a constant you can't overwrite that one. It will keep getting the constant from the original class, unless you override the process method, in which case this doesn't apply.